### ISSUE TITLE
Tag - close button aria-label

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7555]
On the Tag component there is no aria-label on the close icon button and no way for the implementer to provide one.

This relates to WCAG rule 1.1.1 and 1.3.6

**Comment by MikeKingdom** [comment]
The svg child of the button has `aria-label="Close"` Wouldn't that support the screen reader? If not I would think we have a common problem where most icon buttons have an issue.

**Comment by MikeKingdom** [comment]
Good question. Might need some more thought here. I just realized there's not really a way to localize these either in case where we have embedded components/icons which have some form of aria label built in

**Comment by jcfilben** [comment]
Switching this from bug to enhancement. Feels like its not a wcag violation but there is room for improvement here
